src/script/view_model/content/PreferencesAccountViewModel.ts

```diff
@@ -250,6 +253,28 @@ export class PreferencesAccountViewModel {
     }
   };
 
+  changeEmail = async (data: unknown, event: ChangeEvent<HTMLInputElement>): Promise<void> => {
+    try {
+      const enteredEmail = event.target.value;
+
+      await this.userRepository.changeEmail(enteredEmail);
+      amplify.publish(WebAppEvents.WARNING.MODAL, ModalsViewModel.TYPE.ACKNOWLEDGE, {
+        text: {
+          message: 'We have sent you a link to verify your new email',
+          title: 'Check your inbox',
+        },
+      });
+    } catch (error) {
+      this.logger.warn('Failed to send reset email request', error);
+      amplify.publish(WebAppEvents.WARNING.MODAL, ModalsViewModel.TYPE.ACKNOWLEDGE, {
+        text: {
+          message: 'The email you entered is in use or invalid',
+          title: 'Oops...!',
```

Comment:
Will we get translation for this?

Reply:
@lipis please create a placeholder with new texts in the code in the format of:

```ts
          text: {
            message: t('modalIntegrationUnavailableMessage'),
            title: t('modalIntegrationUnavailableHeadline'),
          },
```